msg/VehicleStatus.msg

```diff
@@ -0,0 +1,42 @@
+#
+#   This file is part of MVP-Mission program.
+#
+#   MVP-Mission is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   MVP-Mission is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with MVP-Mission.  If not, see <https://www.gnu.org/licenses/>.
+#
+#   Author: Mingxi Zhou
+#   Email: mzhou@uri.edu
+#   Year: 2022
+#
+#   Copyright (C) 2022 Smart Ocean Systems Laboratory
+#
+
+#global information
+
+string  timenow         #SS-DD-MM-YY
+string  timegps         #SS-DD-MM-YY
```

Comment:
To make it fit for the convention, these should, regardless of its convenience, be std_msgs/Time. I strongly suggest the change in the type.
```suggestion
string  timenow         #SS-DD-MM-YY
string  timegps         #SS-DD-MM-YY
```

https://docs.ros.org/en/noetic/api/std_msgs/html/msg/Time.html